cli/core/config_definition.rs

```diff
@@ -560,3 +576,12 @@ fn default_build() -> BuildConfig {
     with_global_tauri: false,
   }
 }
+
+fn default_updater() -> UpdaterConfig {
```

Comment:
Yes, you are right! The main reason I was using a custom Impl, it's because by default, `dialog = true` 
is it possible to overwrite the field value to true with Derive Default without a custom impl?
or we could rename the field `dialog` for `disable_dialog` or something like that? and default to false will be all good.

Reply:
yeah serde has a default attribute, there's some examples in this file.

Reply:
yeah but the test is failing because the `UpdaterConfig::Default` didn't return the dialog set to true, JSON will work ok with serde deserialization, but not the UpdaterConfig::Default called from tests.

Reply:
yeah you're right :P